cypress/support/elements/common/Canvas.js

```diff
@@ -205,6 +205,26 @@ class Canvas {
       cy.wrap($handle).should('not.have.class', 'selected');
     });
   }
+
+  getFourUpToolbarButton() {
+    return cy.get('[data-testid="tool-fourup"]');
+  }
+
+  getFourUpToolbarButtonState() {
+    return this.getFourUpToolbarButton().invoke('attr', 'class');
+  }
+
+  isFourUpToolbarButtonDisabled() {
+    return this.getFourUpToolbarButtonState().should('contain', 'disabled');
+  }
+
+  isFourUpToolbarButtonEnabled() {
+    return this.getFourUpToolbarButtonState().should('not.contain', 'disabled');
+  }
+
+  clickFourUpToolbarButton() {
+    return this.getFourUpToolbarButton().click();
+  }
```

Comment:
Just a note, you've defined these methods but I don't think you're using them.  The spec files are checking the class attribute and calling click themselves.